src/main/webapp/pages/samples/_base.html

```diff
@@ -34,7 +34,7 @@ <h1 class="t-title" th:text="#{samples.title(${sample.getLabel()})}">__TITLE__</
         <div class="col-md-4 col-md-pull-8">
             <aside class="sidebar">
                 <div class="sidebar__heading">
-                    <h2 id="sample-name" th:text="${sample.getLabel()}">__NAME__</h2>
+                    <h2 class="name_wrap" id="sample-name" th:text="${sample.getLabel()}">__NAME__</h2>
```

Comment:
Everything works exactly as advertised.  One change, can you change the name of this class to `text-wrap` since it can be used to wrap any long text.

Reply:
Yup.  0b47b69